src/kirby/components/floating-action-button/floating-action-button.component.scss

```diff
@@ -5,8 +5,21 @@
 
 ion-fab-button {
     --box-shadow: 0 10px 20px -10px #{get-color('primary-shade')};
-    width: 64px;
-    height: 64px;    
+    
+    &.sm {
+        width: #{size('xl')}px;
```

Comment:
This is the `size` function - note the `addPX`?
```
@function size($key: 'm') {
  $negative: str-index($key, '-') == 1;
  @if ($negative) {
    $key: str-slice($key, 2, -1);
  }
  $value: '';
  @if ($native == false) {
    $value: addPX(_get-map-prop($sizes, $key));
  } @else {
    $value: _get-map-prop($sizes, $key);
  }
  @return if($negative, unquote('-' + $value), $value);
}
```

Reply:
I still haven't gotten this to work, so i kept the explicit px